File: 1-easy/problems/2643-row-with-maximum-ones/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def rowAndMaximumOnes(self, mat):
        """
        :type mat: List[List[int]]
        :rtype: List[int]
        """
        return max(([i, mat[i].count(1)] for i in range(len(mat))), key=lambda x: x[1])

File: 1-easy/problems/2643-row-with-maximum-ones/solution.ts
function rowAndMaximumOnes(mat: any): boolean | number | string | any {
  """
  :type mat: List[List[int]]
  :rtype: List[int]
  """
  return max(([i, mat[i].count(1)] for i in Array.from({length: (mat}, (_, i) => i).length)), key=lambda x: x[1]);
}

export { rowAndMaximumOnes };
